build.yml

```diff
@@ -60,6 +60,30 @@ docker-targets:
       xfonts-base
       zlib1g
 
+  bookworm:
+    source: docker/Dockerfile.debian
+    args:
+      from: debian:bookworm
+      python: python3
+    output: deb
+    matrix: ['amd64', 'arm64']
+    depend: >
+      ca-certificates
+      fontconfig
+      libc6
+      libfreetype6
+      libjpeg62-turbo
+      libpng16-16
+      libssl1.1
```

Comment:
This looks wrong, as [`libssl-dev` depends on libssl3](https://packages.debian.org/bookworm/libssl-dev).